programs/util.h

```diff
@@ -127,6 +127,19 @@ extern int g_utilDisplayLevel;
     typedef struct stat stat_t;
 #endif
 
+int g_excludeCompressedFiles;
```

Comment:
do not declare global variables inside `*.h`, since is a recipe to generate duplicate symbols.